pkg/volume/util/subpath/subpath_windows.go

```diff
@@ -45,30 +45,63 @@ func NewNSEnter(mounter mount.Interface, ne *nsenter.Nsenter, rootDir string) In
 }
 
 // evalPath returns the path name after the evaluation of any symbolic links.
-// If the path after evaluation starts with Volume or \??\Volume, it means that it was a symlink from
-// volume (represented by volumeID) to the given path. In this case, the given path is returned.
+// If the path after evaluation is a device path or network connection, the original path is returned
 func evalPath(path string) (linkedPath string, err error) {
-	cmd := fmt.Sprintf("Get-Item -Path %s | Select-Object -ExpandProperty Target", path)
+	// The following is a workaround solution for issue https://github.com/kubernetes/kubernetes/issues/93759
+	upperpath := path
+	base := ""
+	for {
+		if isDriveLetterorEmptyPath(upperpath) {
+			klog.V(4).Infof("Path '%s' is not a symlink, return its original form.", path)
+			return path, nil
+		}
+		// Check whether a directory is a symliink or not.
+		cmd := fmt.Sprintf("(Get-Item -Path %s).Attributes", upperpath)
+		output, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
+		if err != nil {
+			return "", err
+		}
+		// The above command will give the attributes of a given path, e.g., Directory, ReparsePoint.
+		// It means that the given path is a directory and also a symlink.
+		if strings.Contains(string(output), "ReparsePoint") {
+			break
+		}
+		base = filepath.Join(filepath.Base(upperpath), base)
+		upperpath = filepath.Dir(upperpath)
+	}
+
+	// This command will give the target path of a given symlink
+	cmd := fmt.Sprintf("(Get-Item -Path %s).Target", upperpath)
```

Comment:
What if you have a chain of symlinks?  For example, "a/b/c/d", and `b` and `d` are symlinks?

Reply:
I think the whole function here is first evaluate the lowest layer /d, then it loop through each layer later here https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/subpath/subpath_windows.go#L122.
Windows and Linux share the same logic, I think

Reply:
Discussed offline, the function should make sure the returned path does not have any symlink in it. Use a recursive function to make sure of it.